portfolio/src/main/webapp/index.html

```diff
@@ -13,6 +13,8 @@ <h1>My Portfolio</h1>
       <p>Click here to get a random message:</p>
       <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>
+      <button onclick="getHelloYewOnnUsingArrowFunctions()">HelloYewOnn</button>
```

Comment:
Could it be Hello Yew Onn as more readable for your users?

Reply:
Okay I will fix that